x-pack/plugins/encrypted_saved_objects/server/config.test.ts

```diff
@@ -32,6 +29,17 @@ describe('config schema', () => {
       }
     `);
 
+    expect(ConfigSchema.validate({ encryptionKey: 'z'.repeat(32) }, { dist: true }))
```

Comment:
I know that this is probably "testing the library", but would you mind adding a test to ensure that `encryptionKey` cannot be `null` when `dist: true`? Plugin code checks that the key isn't `undefined`, but if `schema.maybe()` ever allowed `null`, then we would incorrectly assume that an encryption key was set.

Reply:
Good idea, will add, thanks